data/json_files/1e3bff88-29b9-4b3f-be34-bffc38624bac.json

```diff
@@ -6,7 +6,7 @@
       "con-Latn-EC": "103 Singo a'i",
       "en": "Black People",
       "es": "El negro",
-      "_default": "Singo a'i"
+      "_default": "103"
```

Comment:
Same thing here - please undo the change from "Singo a'i" to "103" and the change in date_uploaded